test/integration/users_edit_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class UsersEditTest < ActionDispatch::IntegrationTest
+  def setup
+    @user = users(:michael)
+  end
+
+  test "unsuccessful edit" do
+    log_in_as(@user)
+    get edit_user_path(@user)
+    assert_template 'users/edit'
+    patch user_path(@user), params: { user: { name:  "",
+                                              email: "foo@invalid",
+                                              password:              "foo",
+                                              password_confirmation: "bar" } }
```

Comment:
```suggestion
patch user_path(@user), params: {
  user: { name: '',
          email: 'foo@invalid',
          password: 'foo',
          password_confirmation: 'bar' }
}
```
nits
こちらの方が見やすいかな？
重複しているスペースは消して欲しいですが、その他のインデントはこのままでもokです